src/components/organisms/PageBlueprint/PageHeader.tsx

```diff
@@ -24,9 +24,11 @@ const PageHeader: FC<PageHeaderProps> = ({title, pageTitleAddon, description, lo
           {title} <LoadingOutlined />
         </>
       ) : (
-        <PageTitle>
-          {title} {pageTitleAddon}
-        </PageTitle>
+        <Tooltip title={title}>
```

Comment:
I'm not sure if a tooltip is a proper option. It may look strange, especially for short titles

Reply:
yes, me too. should we enable/disable based on the length of the title?

Reply:
In the best situation we should show it only when it's truncated. Quick googling shows that most likely it's possible, so it would be great if you could try it: https://stackoverflow.com/a/63214232/2922986